db/migrations/20221012105724_init.js

```diff
@@ -9,8 +9,15 @@ exports.up = async function (knex) {
       table.string('first_name', 20).notNullable();
       table.string('last_name', 20).notNullable();
       table.string('email', 30).notNullable();
+      table.integer('phone_number_id', 4).notNullable();
       table.string('password').notNullable();
     });
+
+  await knex.schema
```

Comment:
it's a bad idea to edit migration after creation. For the first time it's ok. I understand, you want to keep first migration clear and simple.